docs/landing_generated/index.html

```diff
@@ -7,6 +7,8 @@
     <title>Envoy: C++ L7 proxy and communication bus</title>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Source+Code+Pro">
     <link href="stylesheets/site-00db0e0a.css" rel="stylesheet" />
+    <link rel="shortcut icon" type="image/png" href="/favicon.ico"/>
```

Comment:
The reason we didn't do this previous is that setting up the ruby code to generate the site downloads a ton of stuff and takes a really long time. Not sure if there is any way to streamline that. I agree that we should automate it. This is good for now, thanks.